code/datums/gamemode/factions/bloodcult/bloodcult_spells.dm

```diff
@@ -175,3 +200,44 @@
 		to_chat(user, "<span class='notice'>You retrace your steps, carefully undoing the lines of the [removed_word] rune.</span>")
 	else
 		to_chat(user, "<span class='warning'>There aren't any rune words left to erase.</span>")
+
+
+//SPELL III
+/spell/cult/blood_dagger
+	name = "Blood Dagger"
+	desc = "Solidify some blood into a sharp weapon. Slash at your enemies to steal their blood. Use the dagger to re-absorb the stolen blood."
+	hud_state = "cult_blooddagger"
+
+	invocation_type = SpI_NONE
+	charge_type = Sp_RECHARGE
+	charge_max = 0
+	range = 0
+	spell_flags = null
+	insufficient_holder_msg = ""
+	still_recharging_msg = ""
+
+	cast_delay = 0
+
+/spell/cult/blood_dagger/choose_targets(var/mob/user = usr)
+	return list(user)
+
+/spell/cult/blood_dagger/cast(var/list/targets, var/mob/living/carbon/user)
+	..()
+	var/mob/living/carbon/human/H = user
+	var/data = use_available_blood(user, 5)
+	if (data[BLOODCOST_RESULT] == BLOODCOST_FAILURE)
+		return 0
+	var/good_hand
+	if(H.can_use_hand(H.active_hand))
+		good_hand = H.active_hand
+	else
+		for(var/i = 1 to H.held_items.len)
+			if(H.can_use_hand(i))
+				good_hand = i
```

Comment:
Did you forget to break somewhere in this ?

Reply:
added